docs/api/algorand/create-transaction-group.mdx

```diff
@@ -0,0 +1,130 @@
+| `txns`      | `Array<TxnObject>`                       | Array containing the description of each transaction to be sent. |
+
+### TxnObject
+
+Each txnObject must have one of the following forms.
+
+|     Name    |                   Type                   | Description |
+| ----------- | ---------------------------------------- | ----------- |
+| `type`      | `"assetTransfer"`                        | Field indicating that this txnObject describes an asset transfer. |
+| `from`      | `string`                                 | The address of the account that is sending the funds. |
+| `to`        | `string`                                 | The address of the account that is receiving the funds. |
+| `asset`     | `string`\|`number`                          | The ID of the asset that is being sent. Can be either a string (i.e. USDD, EURD) or a number (i.e. 51987792, 51987793). |
+| `amount`    | `number`                                 | The amount that is being sent. |
+| `closeTo`   | `string`                                 | OPTIONAL. If set, after sending the funds to the `to` account, sends the rest of the remaining funds to the `closeTo` account. |
+
+
+|     Name    |                   Type                   | Description |
+| ----------- | ---------------------------------------- | ----------- |
+| `type`      | `"algoTransfer"`                         | Field indicating that this txnObject describes an ALGO transfer. |
+| `from`      | `string`                                 | The address of the account that is sending the funds. |
+| `to`        | `string`                                 | The address of the account that is receiving the funds. |
+| `amount`    | `number`                                 | The amount that is being sent. |
+| `closeTo`   | `string`                                 | OPTIONAL. If set, after sending the funds to the `to` account, sends the rest of the remaining funds to the `closeTo` account. |
+
+
+|        Name       |                   Type             | Description |
+| ----------------- | ---------------------------------- | ----------- |
+| `type`            | `"applicationCall"`                | Field indicating that this txnObject describes an application call. |
+| `from`            | `string`                           | The address of the account that is calling the application. |
+| `app`             | `string`                           | The ID of the application that is being called. |
+| `onComplete`      | `"NoOp"`\|`"OptIn"`\|<br/>`"CloseOut"`\|`"Update"`\|`"Delete"` | OPTIONAL. Describes the action that must be done after the call execution. By default is set to `"NoOp"`. |
+| `args`            | `Array<string`\|`number>`             | OPTIONAL. The arguments that must be passed to the application. By default is set to the empty list. |
+| `accounts`        | `Array<string>`                    | OPTIONAL. The accounts arguments. |
```

Comment:
I would describe it as "a list of accounts that this transaction will have access to"

Reply:
Same for descriptions below

Reply:
Yep, might also be good to link some algorand docs